lib/check-node.js

```diff
@@ -2,11 +2,16 @@ const exec = require('child_process').exec
 const semver = require('semver')
 
 module.exports = function (cb) {
-  exec('node --version', (error, stdout, stderr) => {
-    if (error) return (error)
-    var output =  (stdout + stderr).replace(/(\r\n|\n|\r|\ )/gm,"")
-    var sem = semver.valid(output)
+  try {
+    var sem = process.versions.node
```

Comment:
@evanlucas so electron comes with its own node? that's cool. I thought it's just using whatever I had.